multimedia/ffmpeg6/Portfile

```diff
@@ -17,7 +17,7 @@ name                ffmpeg6
 set my_name         ffmpeg
 
 version             6.1.1
-revision            1
+revision            0
```

Comment:
Sergey, was the intent to rev-bump this port, or...?

Reply:
Sorry, this way a mistake, I guess.

Perhaps no need to revbump anything, since it does not build at the moment with non-Rust `librsvg`, and otherwise the change is inconsequential.

Reply:
Fixed.